projects/plugins/jetpack/extensions/blocks/latex/block.json

```diff
@@ -18,7 +18,19 @@
 		}
 	},
 	"supports": {
-		"html": false
+		"html": false,
+		"color": {
+			"gradients": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true
```

Comment:
Does `lineHeight` make sense in `<math>` tag? If it does, can you share some LaTeX to test it with?

Reply:
Not really! Removed.